Classes/Notifications/NotificationsViewController.swift

```diff
@@ -108,18 +148,35 @@ FlatCacheListener {
             // "mark all" is an engaging action, system prompt on it
             RatingController.prompt(.system)
         }
+
+        switch inboxType {
+        case .all, .unread: client.markAllNotifications(completion: block)
+        case .repo(let owner, let name): client.markRepoNotifications(owner: owner, repo: name, completion: block)
+        }
     }
 
-    @objc private func onMarkAll(sender: UIBarButtonItem) {
+    @objc private func onMarkAll() {
+        let message: String
+        switch inboxType {
+        case .all, .unread:
+            message = NSLocalizedString("Mark all notifications as read?", comment: "")
+        case .repo(_, let name):
+            let messageFormat = NSLocalizedString("Mark %@ notifications as read?", comment: "")
```

Comment:
Saw this and my initial thought was "shouldn't this be `%d` to show the number of notifications?" and then I saw that this is for the repo name. What about adding the amount of notifications to this message?

Reply:
It's paged to 50 mixed-repo events, so number here likely wont be accurate if you have more than 50 unread notifications.

Reply:
We \*could\* add the number for < 50 then maybe? 🤔